src/main/java/prefs/GlobalConfig.java

```diff
@@ -28,6 +29,7 @@
     private String lastViewedRepository = "";
     private String lastLoginUsername = "";
     private byte[] lastLoginPassword = new byte[0];
+    private Optional<String> lastOpenBoard;
```

Comment:
Think you might have forgotten to initialise this (to `empty`). Got an NPE when I checked this out and tried to save. It wasn't caught in the test?